benchmark/main.cpp

```diff
@@ -961,8 +964,14 @@ static int benchmark( int argc, char *argv[] )
 }
 
 int main(void) {
-    int ret = benchmark(0, NULL);
+    mbedtls_platform_context platform_ctx;
+    if(mbedtls_platform_setup(&platform_ctx) != 0) {
+        return -1;
```

Comment:
I suggest not covering up the return code here and also adding something like:
```
printf( "Platform initialization failed with error -%04X",  -ret );
```
Not sure what the format should be but thats how we normally print error codes.